modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -149,21 +149,65 @@ const opMakeRowHeader = function (grid: Structs.RowCells[], detail: Structs.Deta
   return bundle(newGrid, detail.row, detail.column);
 };
 
-const opMakeColumnHeader = function (grid: Structs.RowCells[], detail: Structs.DetailExt, comparator: CompElm, genWrappers: GeneratorsTransform) {
-  const newGrid = TransformOperations.replaceColumn(grid, detail.column, comparator, genWrappers.replaceOrInit);
+// Function added to match opMakeColumnsHeader, but not used anywhere.
+const opMakeRowsHeader = function (initialGrid: Structs.RowCells[], details: Structs.DetailExt[], comparator: CompElm, genWrappers: GeneratorsTransform) {
+  const rows = uniqueRows(details);
+
+  const newGrid = Arr.foldl(rows, (currentGrid, row) =>
+    TransformOperations.replaceRow(currentGrid, row.row, comparator, genWrappers.replaceOrInit),
+  initialGrid);
```

Comment:
This is pretty hard to read due to the mixture of the fold and the function wrapping over multiple lines. Maybe something like this instead?
```suggestion
  const replacer = (currentGrid: Structs.RowCells, row: number) => TransformOperations.replaceRow(currentGrid, row.row, comparator, genWrappers.replaceOrInit);
  const newGrid = Arr.foldl(rows, replacer, initialGrid);
```

If you do change this, please replace the 3 other places that are the same as this.

- [ ] Consider using different formatting to make this more readable